csrc/layer_norm_cuda_kernel.cu

```diff
@@ -882,6 +895,7 @@ void HostApplyLayerNorm(
       output, mean, invvar, input, n1, n2, U(epsilon), gamma, beta, warp_size);
 }
 
+// TODO: Optimize HostRMSNormGradient for AMD GPUs: https://github.com/ROCmSoftwarePlatform/apex/pull/66/files
```

Comment:
The TODO commend is not needed any more, since the optimization has been completed in [HostApplyRMSNorm](https://github.com/ROCmSoftwarePlatform/apex/blob/dev/hubertlu/FusedRMSNorm/csrc/layer_norm_cuda_kernel.cu#L906-L932). However, it is for MI100. We will need to test the kernel for MI200 and other hardware in the future.